### DISCUSSION TITLE
How to calculate the difference between the two values of rows?

### DISCUSSION THREAD

**Discussion by BobWs** [Budibase/budibase#14249] [Help]
Hi,
I'm new to Budibase and I'm trying to create a Table for Energie Usage, but I don't get how to calculate the difference between the rows. (See screenshot for how it should works).
![Screen_Shot](https://github.com/user-attachments/assets/e7252b3f-2aa4-4f2c-a269-dfd580daa0d2)

**Comment by ConorWebb96** [comment]
You can use a [formula](https://docs.budibase.com/docs/formula) column. Or in the design area using bindings to total up the values.

In JS it would be something like 

```js
return $("Electra (High) Usage") + $("Electra (Low) Usage")
```
etc. This might be slightly different depending on the columns type. If it is a string you'll have to parse it to an int before writing your JS/handlebars.
